format.go

```diff
@@ -28,33 +28,6 @@ func NewDefaultFormat(withTrace bool) Format {
 	return stringFmt
 }
 
-// UnpackedError represents complete information about an error.
-//
-// This type can be used for custom error logging and parsing. Use `eris.Unpack` to build an UnpackedError
-// from any error type. The ErrChain and ErrRoot fields correspond to `wrapError` and `rootError` types,
-// respectively. If any other error type is unpacked, it will appear in the ExternalErr field.
-type UnpackedError struct {
```

Comment:
this unpacking stuff seems like it belongs in its own file, we can discuss if you don't agree though